google/cloud/_helpers.py

```diff
@@ -42,8 +42,10 @@
 from six.moves.http_client import HTTPConnection
 from six.moves import configparser
 
+# pylint: disable=ungrouped-imports
 from google.cloud.environment_vars import PROJECT
 from google.cloud.environment_vars import CREDENTIALS
+# pylint: enable=ungrouped-imports
```

Comment:
Ugh -- if we have those everywhere, we should maybe just disable the check altogether.